easybuild/easyconfigs/p/Perl/Perl-5.32.1-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,1798 @@
+        'source_tmpl': 'Data-Dump-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/G/GA/GAAS'],
+        'checksums': ['af53b05ef1387b4cab4427e6789179283e4f0da8cf036e8db516ddb344512b65'],
+    }),
+    ('File::Next', '1.18', {
+        'source_tmpl': 'File-Next-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/P/PE/PETDANCE'],
+        'checksums': ['f900cb39505eb6e168a9ca51a10b73f1bbde1914b923a09ecd72d9c02e6ec2ef'],
+    }),
+    ('App::cpanminus', '1.7044', {
+        'source_tmpl': 'App-cpanminus-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/M/MI/MIYAGAWA'],
+        'checksums': ['9b60767fe40752ef7a9d3f13f19060a63389a5c23acc3e9827e19b75500f81f3'],
+    }),
+    ('Parallel::ForkManager', '2.02', {
+        'source_tmpl': 'Parallel-ForkManager-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/Y/YA/YANICK'],
+        'checksums': ['c1b2970a8bb666c3de7caac4a8f4dbcc043ab819bbc337692ec7bf27adae4404'],
+    }),
+    ('Logger::Simple', '2.0', {
+        'source_tmpl': 'Logger-Simple-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/T/TS/TSTANLEY'],
+        'checksums': ['2e63fd3508775b5902132ba1bfb03b42bee468dfaf35dfe42e1909ff6d291b2d'],
+    }),
+    ('Scalar::Util::Numeric', '0.40', {
+        'source_tmpl': 'Scalar-Util-Numeric-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/C/CH/CHOCOLATE'],
+        'checksums': ['d7501b6d410703db5b1c1942fbfc41af8964a35525d7f766058acf5ca2cc4440'],
+    }),
+    ('YAML', '1.30', {
+        'source_tmpl': 'YAML-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/T/TI/TINITA'],
+        'checksums': ['5030a6d6cbffaf12583050bf552aa800d4646ca9678c187add649227f57479cd'],
+    }),
+    ('Object::InsideOut', '4.05', {
+        'source_tmpl': 'Object-InsideOut-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/J/JD/JDHEDDEN'],
+        'checksums': ['9dfd6ca2822724347e0eb6759d00709425814703ad5c66bdb6214579868bcac4'],
+    }),
+    ('Thread::Queue', '3.13', {
+        'source_tmpl': 'Thread-Queue-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/J/JD/JDHEDDEN'],
+        'checksums': ['6ba3dacddd2fbb66822b4aa1d11a0a5273cd04c825cb3ff31c20d7037cbfdce8'],
+    }),
+    ('Time::HiRes', '1.9764', {
+        'source_tmpl': 'Time-HiRes-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/A/AT/ATOOMIC'],
+        'checksums': ['9841be5587bfb7cd1f2fe267b5e5ac04ce25e79d5cc77e5ef9a9c5abd101d7b1'],
+    }),
+    ('Term::ReadLine::Gnu', '1.37', {
+        'source_tmpl': 'Term-ReadLine-Gnu-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/H/HA/HAYASHI'],
+        'checksums': ['3bd31a998a9c14748ee553aed3e6b888ec47ff57c07fc5beafb04a38a72f0078'],
+    }),
```

Comment:
The latest release is 1.40 but after 1.38 build is broken and it complains that it cannot link against -ltinfo but it claims to support readline 8.1.

Reply:
Do I understand this correctly; 1.40 fails to build despite them claiming it works with readline 8.1?

Reply:
The author introduced the support for readline 8.1 with version 1.37.
Changes document:
"1.37  2020-12-27
        - readline-8.1 support".

Starting with 1.38, the linking process no longer works for readline, i.e., it cannot link against -ltinfo. I don't know should we update the easyblock or the author, Hiroo Hayashi, should fix the extension, Term::Readline::GNU.

Reply:
Looking into it more deeply, it seems that it has nothing to do with a particular version of the deadline but rather with the changes in the Makefile.PL. The guy was expanding the support for MacOS, Homebrew, and broke it for us.

Reply:
> @robert-mijakovic Final OpenMPI v4.1.1 release is out now, see https://www.open-mpi.org/software/ompi/v4.1/

TY @boegel